tdp/cli/conftest.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from tdp.conftest import generate_collection
+
+
+@pytest.fixture(scope="function")
```

Comment:
`scope="function"` is default with Pytest. You can remove it.

Reply:
ok